src/apis/finnHub.js

```diff
@@ -1,6 +1,6 @@
 import axios from "axios";
 
-let TOKEN = "cgd20g1r01qum7u5quagcgd20g1r01qum7u5qub0"
+let TOKEN = "cggt9g1r01qv7vi3iom0cggt9g1r01qv7vi3iomg"
```

Comment:
while this doesn't suppose a security problem, bear in mind that tokens, secrets, API keys and so on shouldn't be stored on Github as plain text.

Reply:
mmm good point. I'm not sure how to not show my personal API key and make the project public

Reply:
Maybe I can use the api key from the tutorial's project

Reply:
A simple way to handle this problem is to use environment variables, https://www.twilio.com/blog/working-with-environment-variables-in-node-js-html

that way you could define `TOKEN` as follows:

```js
let TOKEN = process.env.TOKEN
```

then in your computer's console you do:

```bash
export TOKEN="cgd..."
npm run dev
```